packages/victory-core/src/victory-util/domain.js

```diff
@@ -289,23 +289,38 @@ function getDomainWithZero(props, axis) {
   if (propsDomain) {
     return propsDomain;
   }
+
+  const dataset = Data.getData(props);
+  const y0Initial = dataset && dataset[0] ? dataset[0]._y0 : 0;
+  const y0Min = dataset.reduce((min, datum) => {
```

Comment:
Is there a reason for calculating `y0Initial` rather than using `Infinity` as the initial value in this reduce?